### DISCUSSION TITLE
0.9.6 - New languages & New options

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/GetMyIP#50] [Announcements]
#### Changes:
- Added: Slovak language! #44
- Added: German language! #46
- Added: Option to start with Windows #22
- Added: Option to start minimized #22
- Added: Option to minimize to tray (notification area) instead of closing #22
- Added: Option to periodically refresh to keep external IP updated #30
- Added: Option to be notified if IP address changes #30
- Updated: Removed `&#13;` carriage returns from all language files #43
- Updated: All the language files were updated with the latest additions
- Updated: NLog dependency has been updated to the latest version
- Updated: Minor UI tweaks & bug fixes

#### Possible Breaking Change:
Adding the notification mentioned above required changing the build to target windows10.0.19041 or higher. Get My IP **_may no longer work_** on WIndows 7, 8, or Windows 10 with build numbers lower than 19041.

#### Thanks:
This week is the Thanksgiving 🦃 holiday in the US. **I'm thankful** for**everyone** that contributed new language translations and to those that are keeping them updated! 🙏🏻

If you wish to contribute a translation, check out the [Contribute a Translation](https://github.com/Timthreetwelve/GetMyIP/wiki/Contribute-a-Translation) topic in the Wiki. Your help would mean the world to me!

**Full Changelog**: https://github.com/Timthreetwelve/GetMyIP/compare/v0.9.3...v0.9.6

## New Contributors
* @uDEV2019 made their first contribution in https://github.com/Timthreetwelve/GetMyIP/pull/46

#### Checksums :
|File|SHA256 hash|
|----|---------------|
|Installer (exe)| `57aee119062225c1714fd623465c9e32e141f04a3b7242f61878360148bb841e` |
|Portable (zip)| `974c7eeb358c216a89434d34cef0fdff6c3fb139a2241cd720477b286abd60b0` |

<hr /><em>This discussion was created from the release <a href='https://github.com/Timthreetwelve/GetMyIP/releases/tag/v0.9.6'>0.9.6 - New languages & New options</a>.</em>

**Comment by CMTriX** [comment]
Where is the setup :p

**Reply by Timthreetwelve** [reply]
Oops!  The files are there now. It might take a few minutes for them to show up.

Thanks for having my back!
